### PR TITLE
Addition of Hybrid Data Manager SDK

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.reference.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HybridData</PackageId>
+    <Description></Description>
+    <Version>0.9.0-preview</Version>
+    <AssemblyName>Microsoft.Azure.Management.HybridData</AssemblyName>
+    <PackageTags>Management;Storage;StorSimple;StorSimpleDataManager;</PackageTags>
```
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please fix the tags

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <PackageReleaseNotes>
+      <![CDATA[
+      This is the first public release of the Azure StorSimple Data Manager SDK.
```
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Is this hybrid or storsimple?

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please fix the tests too

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj (hunk 2 above)
Comment (t-prta) [reply src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: HybridDataManager was the old name for the service. The public facing name is StorSimple Data Manager Service

Comment (dsgouda) [reply src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Then you should be creating a directory with its appropriate name. What is the RP name now?

Comment (arbasu) [reply src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Hi, The resource provider namespace is "Microsoft.HybridData", but in public documentation, or in releases its mentioned as StorSimple Data Manager. [public doc](https://docs.microsoft.com/en-us/azure/storsimple/storsimple-data-manager-overview). Hence, we use the HybridDataManager. Let us know if this is a problem, in that case, what should be the name.

**Comment by t-prta** [comment]
Hi, 
The tests require me to set some customer secrets. These are passed as Environment variables for the tests, and as the build for the PR does not set these values, some of the tests are failing. I am attaching the test results from my system with all tests passing. The command I ran was 
msbuild build.proj /t:RunTests /p:Scope=SDKs\HybridDataManager
and the result file is below
[HybridData.Tests.txt](https://github.com/Azure/azure-sdk-for-net/files/3071505/HybridData.Tests.txt)

**Comment by t-prta** [comment]
The swagger spec PR is merged. [PR link](https://github.com/Azure/azure-rest-api-specs/pull/5313)

**Comment by dsgouda** [comment]
@t-prta Please regenerate the code using spec that was merged (generate.ps1)

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**